src/index.html

```diff
@@ -5,9 +5,41 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+      class="card__img"
+      src="./images/imac.jpeg"
+      alt="screen image"
+      >
+
+      <p class="card__content">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+
+      <p class="card__product-code">Product code: 195434</p>
+
+      <div class="stars">
+        <div class="stars__box">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="stars__reviews">Reviews: 5</p>
+      </div>
```

Comment:
according to BEM recommendations
```suggestion
      <div class="card__rating">
        <div class="card__stars stars">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>

        <p class="card__reviews">Reviews: 5</p>
      </div>
```